api/getBlogs/data/VaranasiAyodhyaAllahbad.json

```diff
@@ -0,0 +1,63 @@
+{
+    "title": "Embark on a Spiritual Journey: The Ultimate Varanasi, Ayodhya, and Allahabad Tour Package Guide",
```

Comment:
File name is Varanasi, Ayodhya, and Allahabad Tour Package Guide. See title and H1 to find page name/route

Reply:
149 KB